src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -22,15 +25,15 @@
       || request.getSession().getAttribute("user").equals("GavinLifrieri")) { %>
        <a href="/admin">Administration</a>
         <% } else{ %>
-          <a href="/login">Login</a>
+          <a href="/login">Log Out</a>
```

Comment:
Does this actually log the user out or does it just take them back to the login page? (If they then go back to the conversations page, are they still logged in?)

Reply:
It just shows them log in page, I thought it was weird it said Log in if you were already logged in